.github/workflows/contracts.yml

```diff
@@ -319,32 +335,24 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
-      - uses: google-github-actions/setup-gcloud@v0.2.0
+      - name: Get upstream packages' versions
+        uses: keep-network/upstream-builds-query@v1
+        id: upstream-builds-query
         with:
-          project_id: ${{ env.GOOGLE_PROJECT_ID }}
-          service_account_key: ${{ secrets.KEEP_TEST_GCR_JSON_KEY }}
-
-      - name: Fetch external contracts artifacts
-        env:
-          CONTRACT_DATA_BUCKET_DIR: keep-core-celo
-          ETH_NETWORK_ID: ${{ env.NETWORK_ID }}
-        run: ./scripts/ci-provision-external-contracts.sh
+          upstream-builds: ${{ github.event.inputs.upstream_builds }}
+          query: |
+            keep-core-solidity-version = github.com/keep-network/keep-core/solidity#version
```

Comment:
We should be using celo specific package here. As we're not yet publishing it we can revert the changes to the previous solution with bucket and leave a todo for the future.